client/src/Poem.jsx

```diff
@@ -10,11 +12,16 @@ PoemLine.propTypes = {
   line: PropTypes.string,
 };
 
+const NBSP = '\u00A0'; // Unicode value for non-breaking space
+
+PoemLine.defaultProps = {
+  line: NBSP,
+};
+
 const Poem = ({ poem }) => {
-  const poemLineNodes = poem.text.split(/\n/).map((line, index) => {
-    const NBSP = '\u00A0'; // Unicode value for non-breaking space
-    return <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={index} />;
-  });
+  const poemLineNodes = poem.text.split(/\n/).map(line =>
+    <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={_.uniqueId()} />,
```

Comment:
This was inspired by [the rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md) "Prevent usage of Array index in keys".

Since each line doesn't have a natural id (vs., e.g., a poem), it made sense to attach some random id, and I found this as a suggestion for such. Perhaps there's a more natural way to accomplish this?